src/lib/pages/backoffice/Series/SeriesDetails/SeriesMetadata/SeriesMetadata.js

```diff
@@ -29,6 +31,20 @@ export default class SeriesMetadata extends Component {
         name: 'Publication Year',
         value: seriesDetails.metadata.publication_year,
       },
+      {
+        name: 'Keywords',
+        value: <DocumentKeywords keywords={seriesDetails.keywords} />,
+      },
+      {
+        name: 'Tags',
+        value: (
+          <DocumentTags
+            isBackOffice
+            size="mini"
+            metadata={seriesDetails.metadata}
```

Comment:
It was not you creating this, but would it be possible to fix also this one and pass only the tags instead?

Reply:
Sure thing, there is at least one more component using this pattern (that's where I took the inspiration from). Will fix all of them.